resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java

```diff
@@ -123,13 +118,14 @@ public Map<String, String> getCommonTemplateVariables(ResidentTransactionEntity
 		templateVariables.put(TemplateVariablesConstants.PURPOSE, residentTransactionEntity.getPurpose());
 		templateVariables.put(TemplateVariablesConstants.ATTRIBUTE_LIST, getAttributesDisplayText(
 				replaceNullWithEmptyString(residentTransactionEntity.getAttributeList()), languageCode, requestType));
-		String authenticationMode = getAuthTypeCodeTemplateValue(
-				replaceNullWithEmptyString(residentTransactionEntity.getAuthTypeCode()), languageCode);
-		if (authenticationMode.equalsIgnoreCase(UNKNOWN)) {
-			templateVariables.put(TemplateVariablesConstants.AUTHENTICATION_MODE,
-					replaceNullWithEmptyString(residentTransactionEntity.getAuthTypeCode()));
+		String authTypeCode = replaceNullWithEmptyString(residentTransactionEntity.getAuthTypeCode());
+		AuthenticationModeEnum authenticationMode = AuthenticationModeEnum.getAuthenticationModeName(authTypeCode, env);
```

Comment:
Whether multiple comma separated Authentication Modes in the authTypeCode is handled?

Reply:
I am working on it...

Reply:
done